source/system/native/MutexUnix.ooc

```diff
@@ -19,7 +19,17 @@ version(unix || apple) {
 			super()
 		}
 		lock: override func {
-			pthread_mutex_lock(this _backend&)
+			version(debugDeadlock) {
+				timeLimitMilliseconds := 1000 //Only whole seconds
```

Comment:
Well, to not overcomplicate things as we lack a `TimeSpec + operator`. If you believe we might need fractions of a second in the future I'm happy to add it.